functions/lib/utils/firebase_sync_event_ticket_types.py

```
@@ -0,0 +1,316 @@
+
+    for collection_path in ACTIVE_EVENT_PATHS:
+        docs = db.collection(collection_path).stream()
+        for doc in docs:
+            event_id = doc.id
+            try:
+                data = doc.to_dict() or {}
+                plan = _plan_event_sync(event_id, collection_path, data)
+                if plan["action"] in ("create_ticket_type", "update_ticket_type") and not dry_run:
+                    doc.reference.update({"eventTicketTypes": plan["eventTicketTypes"]})
+                _accumulate_plan(plan, buckets)
+            except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)